pkg/corerp/frontend/controller/applications/testdata/graph-app-gw-out.json

```diff
@@ -0,0 +1,53 @@
+[
```

Comment:
@nithyatsu @rynowak Please review this appgraph output carefully. To me, the result makes sense.

Reply:
Looks right to me 👍